<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SLC_deramp</class>
 <widget class="QWidget" name="SLC_deramp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>去参考相位</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../QtWidgetsApplication3.qrc">
    <normaloff>:/QtWidgetsApplication3/bin/icon/dem.png</normaloff>:/QtWidgetsApplication3/bin/icon/dem.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,7">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>目标工程</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,7">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>数据节点</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_dst_node"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,7">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>目标节点名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../QtWidgetsApplication3.qrc"/>
 </resources>
 <connections/>
</ui>
